docs/README.iOS.md

```diff
@@ -126,6 +131,34 @@ Build a specific group of add-ons:
 make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons ADDONS="pvr.*"
 ```
 
+## 5.2. Xcode project building
+
+Binary addons will be built as a dependency in the Xcode project. You can choose the addons you wish to build during the Xcode project generation step
```

Comment:
Maybe add line breaks here and below to have better readable in file. For Markdown it still stays in one line then.

Reply:
Have added line breaks. Easier to review as suggested.